src/App.js

```diff
@@ -14,11 +15,96 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGood: ['Jam'],
+  };
+
+  componentDidUpdate() {
+    document.querySelectorAll('li').forEach((element) => {
+      if (this.state.selectedGood.includes(element.textContent)) {
+        element.classList.add('active');
+      } else {
+        element.classList.remove('active');
+      }
+    });
+
+    document.querySelectorAll('.clearAll-button').forEach((element) => {
+      if (this.state.selectedGood.length === 0) {
+        element.classList.add('hover');
+      } else {
+        element.classList.remove('hover');
+      }
+    });
+  }
+
+  selectedGoods = (selectedGood) => {
```

Comment:
```suggestion
  getSelectedGoodsTitle = (selectedGood) => {
```
Let's use more meaningful names. It's better to name the functions according to what they actually do